examples/ida_binaryai.py

```diff
@@ -59,12 +65,26 @@ def retrieve_selected_functions(self, funcs):
                 print("[{}] {} is skipped because top1_score lower than threshold({})".format(
                     self.name, func_name, self.cfg['threshold']))
                 continue
-            pfn.flags |= idaapi.FUNC_LUMINA
-            idaapi.update_func(pfn)
+            idc.set_color(ea, idc.CIC_FUNC, self.get_binaryai_function_color())
```

Comment:
this line will also set the background color in the IDA-View window, which is not intended.
we prefer to find a better way, the same way as lumina did.